app/cisco/ccna/beginner-guide/page.css

```
@@ -0,0 +1,344 @@
+    padding: 1rem 1.25rem;
+    margin: 0 0 1.6rem;
+    color: var(--ccna-text-dim);
+    font-size: 0.94rem;
+}
+.ccna-beginner-page .callout strong {
+    color: var(--ccna-accent-strong);
+}
+.ccna-beginner-page .callout.warn {
+    border-left-color: var(--ccna-warn);
+    background: var(--ccna-warn-bg);
+    border-color: var(--ccna-warn-border);
```

Stylelint (17.14.0)

[error] Overridden property "border-left-color" by shorthand "border-color" (declaration-block-no-shorthand-property-overrides)

(declaration-block-no-shorthand-property-overrides)